src/RecursivePublishable.php

```diff
@@ -409,7 +410,15 @@ public function unlinkDisownedRelationship($source, $targetStage, $relationship)
         }
 
         // Query the source for the list of items to NOT remove
-        $ownedSQL = $owner->getComponents($relationship)->sql($ownedParams);
+        $ownedSQL = $owner->getComponents($relationship);
```

Comment:
Maybe rename this variable until you actually get it converted to an SQL string. Otherwise, you'll have a variable that start as a `DataList` and then gets assigned a string value later on.

```suggestion
        $ownedQuery = $owner->getComponents($relationship);
```